README.md

```diff
@@ -2,37 +2,38 @@
-
-### If you're applying for a full stack role
+### If you are applying for a backend role
+* Render the list of stores from the stores.json file in alphabetical order using a template.
+* Use postcodes.io to get the latitude and longitude for each postcode. Render them next to each store location in the template.
+* Build the functionality that allows you to return a list of stores in a given radius of a given postcode in the UK. The list must be ordered from north to south. No need to render anything, but the function needs to be unit tested.
 
-* Build an API that returns stores from the `stores.json` file, based on a given search string and unit test it, i.e. return "Newhaven" when searching for "hav" - make sure the search allows to use both city name and postcode
-* Order the results by matching postcode first and then matching city names, i.e. "br" would have "Orpington" as the 1st result (as its postcode is "BR5 3RP"), and then "Bracknell", "Broadstairs", "Tunbridge_Wells", and "Brentford"
-* Using your favourite frontend framework (we'd prefer Vue) on the user-facing side
-* Build a frontend that renders a text field for the query and the list of stores that match it
-* Add suggestions to the query field as you type, with a debounce effect of 100ms and a minimum of 2 characters
-* Limit the results to 3 and lazy load the rest on page scroll
+### If you are applying for a full stack role
+* Build an API that returns stores from the `stores.json` file, based on a given search string and unit test it. For example, return "Newhaven" when searching for "hav". Make sure the search allows to use both city name and postcode.
+* Order the results by matching postcode first and then matching city names. For example, "br" would have "Orpington" as the 1st result as its postcode is "BR5 3RP". Next would be "Bracknell", "Broadstairs", "Tunbridge_Wells", and "Brentford"
+* Using your favourite frontend framework (we would prefer Vue) on the user-facing side:
+  * Build a frontend that renders a text field for the query and the list of stores that match it
+  * Add suggestions to the query field as you type, with a debounce effect of 100ms and a minimum of 2 characters
+  * Limit the results to 3 and lazy load the rest on page scroll
 
 ### Finally
 
-* Zip your code up and upload it into Greenhouse (our recruitment system) using the link provided at the bottom of the email you received
-* Tell us which test you completed (backend or full-stack)
-* Tell us what you would have done differently if you'd have had more time?
-* What bits did you find the toughest? What bit are you most proud of? In both cases, why?
-* What's one thing we could do to improve this test?
+* Zip your code up and upload it to Greenhouse (our recruitment system). Use the link provided at the bottom of the email you received.
```

Comment:
"email that you received" ?

Reply:
> "email that you received" ?

If, "you received.", was something like, "you received during your time locked up in a Cambodian prison.", I'd probably include "that", but considering the simplicity it's intelligible both ways.

Reply:
Yeah, I'm really proposing it as it reads more nicely to me. No need for it to be a blocker at all, just a suggestion really :)

Reply:
> Yeah, I'm really proposing it as it reads more fluidly to me. No need for it to be a blocker at all, just a suggestion really :)

Ye "that" is probably the most awkward word to use in English. I often catch myself either over using it or omitting it too much.